scene/gui/control.cpp

```diff
@@ -1777,7 +1777,7 @@ Rect2 Control::_compute_child_rect(const float p_anchors[4], const float p_margi
 	Rect2 anchorable = get_parent_anchorable_rect();
 	Rect2 result = anchorable;
 	for (int i = 0; i < 4; i++) {
-		result.grow_margin((Margin)i, p_anchors[i] * anchorable.get_size()[i % 2] + p_margins[i]);
+		result = result.grow_margin((Margin)i, p_anchors[i] * anchorable.get_size()[i % 2] + p_margins[i]);
```

Comment:
That's likely what was intended indeed, but it will change the logic. CC @groud to check that it properly fixes a bug or if that unknown bug was worked around further down.

Reply:
Well, in fact this function could be removed completely, it is not used in the code anywhere.